prost-types/README.md

```diff
@@ -16,6 +16,6 @@ included under its original ([BSD][2]) license.
 
 [2]: https://github.com/google/protobuf/blob/master/LICENSE
 
-See [LICENSE](..LICENSE) for details.
+See [LICENSE](../LICENSE) for details.
```

Comment:
That will fix the problem in the repo, but that will not work on [crates.io](https://crates.io/crates/prost-types). As there, only the content of the `prost-types` directory is available. Luckily, the `LICENSE` file has a symlink next to the readme.

```suggestion
See [LICENSE](./LICENSE) for details.
```